src/display/editor/editor.js

```diff
@@ -1319,9 +1389,138 @@ class AnnotationEditor {
     if (this.isResizable) {
       this.#createResizers();
       this.#resizersDiv.classList.remove("hidden");
+      bindEvents(this, this.div, ["keydown"]);
     }
   }
 
+  /**
+   * onkeydown callback.
+   * @param {KeyboardEvent} event
+   */
+  keydown(event) {
+    if (
+      !this.isResizable ||
+      event.target !== this.div ||
+      event.key !== "Enter"
+    ) {
+      return;
+    }
+    this._uiManager.setSelected(this);
+    this.#savedDimensions = {
+      savedX: this.x,
+      savedY: this.y,
+      savedWidth: this.width,
+      savedHeight: this.height,
+    };
+    const children = this.#resizersDiv.children;
+    if (!this.#allResizerDivs) {
+      this.#allResizerDivs = Array.from(children);
+      for (const div of children) {
```

Comment:
Would it perhaps be a tiny bit more efficient to only access `this.#resizersDiv.children` *once* here, since that requires querying the DOM?
```suggestion
    if (!this.#allResizerDivs) {
      this.#allResizerDivs = Array.from(this.#resizersDiv.children);
      for (const div of this.#allResizerDivs) {
```

Unless I'm misunderstanding things, it seems that you could just iterate through `this.#allResizerDivs` in all the relevant `for`-loops below.